webpack.config.js

```diff
@@ -29,6 +30,11 @@ module.exports = {
       }
     ]
   },
+  plugins: [
+    new webpack.EnvironmentPlugin([
+      'CACHE_IFRAME_URL',
+    ])
+  ],
```

Comment:
This isn't needed if we set it it `config.js`

Reply:
Noted